python/src/main/java/org/apache/zeppelin/python/PythonInterpreter.java

```diff
@@ -464,12 +467,23 @@ public int getProgress(InterpreterContext context) throws InterpreterException {
   public List<InterpreterCompletion> completion(String buf, int cursor,
                                                 InterpreterContext interpreterContext)
       throws InterpreterException {
+
+    final List <InterpreterCompletion> completions = LSPUtils.
+        getLspServerComplitions(buf, cursor,
```

Comment:
this doesn;t seem like the style for multi - line code
```
LSPUtils.
getLspServerComplitions
```

also `Complitions`?

Reply:
Fixed